packages/zone/src/virtual.js

```diff
@@ -25,7 +24,7 @@ const initEmpty = harden(() => emptyRecord);
  *
  * @type {import('.').Zone['exo']}
  */
-const defineVirtualExo = (
+const makeVirtualExo = (
```

Comment:
The `make` vs `define` convention was introduced to distinguish functions that make things vs functions that define and return make functions. This name change is more consistent with the matrix this is completing.